### DISCUSSION TITLE
Add (see all) link for Most Downloaded, New Crates, etc

### DISCUSSION THREAD

**Discussion by eval-exec** [rust-lang/crates.io#5403] [General]
In the homepage of https://crates.io, we only can view top 10 New Crates, or top 10 Most Downloaded crates, I think it's would be awesome to show more crates instead of top 10.

![image](https://user-images.githubusercontent.com/46400566/199457606-8acd50a5-8dc2-420e-8e5b-f53fa75e1892.png)

**Comment by ToBinio** [comment]
are there any plans to implement this request or is there some reason not to?

**Reply by Turbo87** [reply]
I don't think anyone had time for it yet

**Comment by ToBinio** [comment]
This discussion got resolved in #6442
So i belive it could be closed

**Comment by eval-exec** [comment]
Thank you :blue_heart:
